src/KubeLibrary/KubeLibrary.py

```diff
@@ -359,14 +360,14 @@ def assert_pod_has_labels(self, pod, labels_json):
             for k, v in labels.items():
                 if pod.metadata.labels and k in pod.metadata.labels:
                     if pod.metadata.labels[k] != v:
-                        logger.error(f'Label "{k}" value "{v}" not matching actual "{pod.metadata.labels[k]}"')
+                        logger.error('Label "{k}" value "{v}" not matching actual "{pod.metadata.labels[k]}"')
```

Comment:
Please leave those f-string as they are. Otherwise the variables won't get resolved in the string output.
https://docs.python.org/3.8/reference/lexical_analysis.html#f-strings

Reply:
Done.